oluseyianani/stackoverflow-lite
# #159947078 EsLint Airbnb Style guide setup applied

#### What does this PR do?
Checks the code base and applies airbnb as the specified style guide.
#### Description of Task to be completed?
Make sure that code meets Airbnb Style guide.
#### How should this be manually tested?
After cloning the repo, cd into it, and RUN `npm install` to make sure eslint and 
other eslint dependencies are installed.
#### Any background context you want to provide?
You have to have Eslint installed to check if it works.
#### What are the relevant pivotal tracker stories?
#159947078
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
